pallets/admin-utils/src/lib.rs

```diff
@@ -852,4 +874,6 @@ pub trait SubtensorInterface<AccountId, Balance, RuntimeOrigin> {
     fn set_weights_set_rate_limit(netuid: u16, weights_set_rate_limit: u64);
     fn init_new_network(netuid: u16, tempo: u16);
     fn set_weights_min_stake(min_stake: u64);
+    fn set_nominator_min_required_stake(min_stake: u64);
+    fn set_minimum_staking_threshold(min_stake: u64);
```

Comment:
What's the difference between the minimum required stake and the minimum staking threshold?